crates/bevy_ui/src/node_bundles.rs

```diff
@@ -310,13 +347,9 @@ pub struct ButtonBundle {
     /// Whether this node should block interaction with lower nodes
     pub focus_policy: FocusPolicy,
     /// The background color, which serves as a "fill" for this node
-    ///
-    /// When combined with `UiImage`, tints the provided image.
     pub background_color: BackgroundColor,
     /// The color of the Node's border
     pub border_color: BorderColor,
-    /// The image of the node
-    pub image: UiImage,
```

Comment:
Why remove this component for buttons ?

Reply:
I believe my thought process was that it's strange to have both `BackgroundColor` and `UiImage` competing for the use case of "background of the button" in the same bundle. You may want to spawn a button that only has a solid color background, but no image, for example.

Reply:
I believe my thought process was that it's strange to have both `BackgroundColor` and `UiImage` competing for the use case of "background of the button" in the same bundle. You may want to spawn a button that only has a solid color background, but no image, for example.

Reply:
Yeah, more specifically, after this PR:

- If you want a button with a solid colored background, you need to set the `BackgroundColor` and have no `UiImage` component.
- If you want a button with an image background, you need to set the `BackgroundColor` to `LegacyColor::NONE` and set the `UiImage` component.

Reply:
This is IIRC, I'll go test it to make sure.

Reply:
I don't think adding a breaking change, here removing a component from `ButtonBundle` should be part of the PR. `Handle<Image>` is, imo, much more likely to be used than `BackgroundColor` as most UI use cases would prefer textured buttons instead of just colored ones

Reply:
I agree, we can have a follow up removing the component where it doesn't make sense, but maybe removing this now is out of scope IMO